source/benchmarking/benchmarking.md

```diff
@@ -611,12 +725,15 @@ TBD: spec system to hold scores over time
 TBD: generated datasets should be park in S3 or somewhere for retrieval by URL
 
 ## Changelog
+- 2024-11-19:
+    - Add Client and Collection BulkWrite benchmarks
+    - Use inline code syntax for all text in single quotes.
```

Comment:
This entry is causing the lint check to fail: https://github.com/mongodb/specifications/actions/runs/12436624595/job/34724832915?pr=1733

We can resolve it by removing the nested list (no need to mention the single quote change):
```
- 2024-12-XX: Add Client and Collection BulkWrite benchmarks

```

Reply:
Thanks, I've updated it but the lint check is still failing. I am not sure as to what causes the error

Reply:
Do you have pre-commit set up locally? https://github.com/mongodb/specifications?tab=readme-ov-file#linting